package.json

```diff
@@ -83,12 +83,12 @@
   "devDependencies": {
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "@bigcommerce/eslint-config": "^1.0.0",
-    "@bigcommerce/tslint-config": "^4.0.1",
+    "@bigcommerce/eslint-config": "^1.0.1",
     "@faker-js/faker": "^6.3.1",
     "@nrwl/cli": "^13.10.3",
-    "@nrwl/eslint-plugin-nx": "^14.3.6",
+    "@nrwl/eslint-plugin-nx": "^13.10.3",
```

Comment:
Why are we downgrading the plugin version?

Reply:
Because `@nrwl/****` should match the version with `nx`. We are on `nx` 13.x, not the lastest 14. I learned this when I came across a strange error message.